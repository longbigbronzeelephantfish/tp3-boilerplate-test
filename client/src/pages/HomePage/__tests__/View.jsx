import React from 'react';
import TestRenderer from 'react-test-renderer';

import HomePageView from 'pages/HomePage/View';

describe('NotesPage tests', () => {
  test('The NotesPage renders without any error', () => {
    const component = TestRenderer.create(<HomePageView datetime="Hello" />);

    let tree = component.toJSON();

    tree.props.datetime = 'Hello';

    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
