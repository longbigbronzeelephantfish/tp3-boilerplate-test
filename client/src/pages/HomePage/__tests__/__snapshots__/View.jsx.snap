// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotesPage tests The NotesPage renders without any error 1`] = `
<div>
  <h1
    className="ui header"
  >
    Hello
  </h1>
  <p>
    The time now is: 
    Hello
    .
  </p>
</div>
`;
